CONTRIBUTING.md

```diff
@@ -105,6 +111,17 @@ This patch eliminates the race condition in issue #1234.
 Issue: #1234
 ```
 
+However, if this is the final commit that addresses the issue then the following
```

Comment:
I would prefer less strongly worded text, similar to `However, if this is the final commit that addresses the issue the following is a valid commit message`.

Reply:
If the Author of a commit put the `Issue:` tag in the commit message the Committer who is merging the commit can always modify the merge commit message to include the `closes` or `fixes`, etc. tag.

Reply:
Added your less strongly worded text.  Your second comment is a true statement, but is it something you want in the CG?

Reply:
No second comment was just a comment :).  I agree it would not make sense in the CG.